readthedocs/gold/templates/gold/projects.html

```diff
@@ -4,53 +4,52 @@
 
 {% block profile-admin-gold-edit %}active{% endblock %}
 
-{% block title %}
-Gold Projects
-{% endblock %}
+{% block title %}{% trans "Gold Projects" %}{% endblock %}
 
 
 {% block edit_content %}
 
-  <p class="empty">
+  <h3>{% trans "Gold Projects" %}</h3>
+  <p>
     {% blocktrans trimmed %}
-      <strong>Note:</strong> this is a legacy feature.
-      New gold members cannot sponsor projects to be ad-free.
+      Gold Members may completely remove advertising for all visitors to their projects.
     {% endblocktrans %}
   </p>
 
-  <p class="help_text">
-    {% trans "Choose projects that will have all promos removed, and extra features added to them. You get to pick one for every $5/mo you support Read the Docs with." %}
-  </p>
-
-  <h3> {% trans "Existing Projects" %} </h3>
-  <p>
-    {% blocktrans trimmed count projects=gold_user.num_supported_projects %}
-      You can adopt one project with your subscription.
-    {% plural %}
-      You can adopt {{ projects }} projects with your subscription.
-    {% endblocktrans %}
-  </p>
+  <ul class="donate-about">
+    <li>{% trans 'For small businesses or personal projects, we ask for $5 per month to remove ads from a project.' %}</li>
+    <li>{% trans 'For corporate supported open source projects, we ask for a $50 membership in order to cover our support and operations costs.' %}</li>
+  </ul>
 
-  <ul>
-  {% for project in projects %}
+  <h3>{% trans 'Adopted projects' %}</h3>
```

Comment:
Should probably just call these ad free. 
```suggestion
  <h3>{% trans 'Ad-free projects' %}</h3>
```